packages/backend/server/schema.prisma

```diff
@@ -480,12 +480,15 @@ model AiSessionMessage {
 }
 
 model AiSession {
-  id          String   @id @default(uuid()) @db.VarChar(36)
-  userId      String   @map("user_id") @db.VarChar(36)
-  workspaceId String   @map("workspace_id") @db.VarChar(36)
-  docId       String   @map("doc_id") @db.VarChar(36)
-  promptName  String   @map("prompt_name") @db.VarChar(32)
-  createdAt   DateTime @default(now()) @map("created_at") @db.Timestamptz(6)
+  id          String    @id @default(uuid()) @db.VarChar(36)
+  userId      String    @map("user_id") @db.VarChar(36)
+  workspaceId String    @map("workspace_id") @db.VarChar(36)
+  docId       String    @map("doc_id") @db.VarChar(36)
+  promptName  String    @map("prompt_name") @db.VarChar(32)
+  messageCost Int       @default(0)
+  tokenCost   Int       @default(0)
+  createdAt   DateTime  @default(now()) @map("created_at") @db.Timestamptz(6)
+  deletedAt   DateTime? @map("deleted_at") @db.Timestamptz(6)
```

Comment:
delete it directly?

Reply:
we need keep session cost to statistics the usage, will use to limit quota